operator/informer_customcache.go

```diff
@@ -48,19 +48,19 @@ type CustomCacheInformer struct {
 // NewMemcachedInformer creates a new CustomCacheInformer which uses memcached as its custom cache.
 // This is analogous to calling NewCustomCacheInformer with a MemcachedStore as the store.
 func NewMemcachedInformer(kind resource.Kind, client ListWatchClient, namespace string, addrs ...string) (*CustomCacheInformer, error) {
-	return NewMemcachedInformerWithLabelFilters(kind, client, namespace, nil, addrs...)
+	return NewMemcachedInformerWithLabelFilters(kind, client, ListWatchOptions{Namespace: namespace}, addrs...)
 }
 
 // NewMemcachedInformerWithLabelFilters creates a new CustomCacheInformer which uses memcached as its custom cache.
 // This is analogous to calling NewCustomCacheInformer with a MemcachedStore as the store.
-func NewMemcachedInformerWithLabelFilters(kind resource.Kind, client ListWatchClient, namespace string, labelFilters []string, addrs ...string) (*CustomCacheInformer, error) {
+func NewMemcachedInformerWithLabelFilters(kind resource.Kind, client ListWatchClient, filterOptions ListWatchOptions, addrs ...string) (*CustomCacheInformer, error) {
```

Comment:
I think this should be renamed to `NewMemcachedInformerWithFilters`, as `WithLabelFilters` is not longer correctly descriptive.

Reply:
Updated.